core/cat/routes/plugins.py

```diff
@@ -28,7 +28,18 @@ async def get_available_plugins(
     # index registry plugins by url
     registry_plugins_index = {}
     for p in registry_plugins:
-        plugin_url = p["url"]
+        plugin_url = p.get("plugin_url", None)
+        if plugin_url is None:
+            log.warning(f"Plugin {p.get('name')} has no `plugin_url`. It will be skipped from the plugins list.")
```

Comment:
from the `registry list`, because locally you can have plugins without manifest (less friction for devx).